apps/prairielearn/src/pages/instructorAssessmentStatistics/instructorAssessmentStatistics.html.ts

```diff
@@ -0,0 +1,333 @@
+                                return JSON.stringify(label);
+                              })}`,
+                            )},
+                          ],
+                        };
+                        histogram('#durationHist', data, xgrid, options);
+                      });
+                    </script>
+                  </div>
+
+                  <div class="table-responsive">
+                    <table class="table table-sm table-hover">
+                      <tbody>
+                        <tr>
+                          <td>Mean duration</td>
+                          <td>${durationStat.mean}</td>
+                        </tr>
+                        <tr>
+                          <td>Median duration</td>
+                          <td>${durationStat.median}</td>
+                        </tr>
+                        <tr>
+                          <td>Minimum duration</td>
+                          <td>${durationStat.min}</td>
+                        </tr>
+                        <tr>
+                          <td>Maximum duration</td>
+                          <td>${durationStat.max}</td>
+                        </tr>
+                      </tbody>
+                    </table>
+                  </div>
+                  <div class="card-footer">
+                    <small>
+                      Download
+                      <a
+                        href="${resLocals.urlPrefix}/assessment/${resLocals.assessment
+                          .id}/assessment_statistics/${resLocals.durationStatsCsvFilename}"
+                        >${resLocals.durationStatsCsvFilename}</a
+                      >. Data outside of the plotted range is included in the last bin.
+                    </small>
+                  </div>
+                `
+              : html`<div class="card-body">No student data.</div>`}
+          </div>
+
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">
+              ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Duration versus score
+            </div>
+
+            ${resLocals.assessment.score_stat_number > 0
+              ? html`
+                  <div class="card-body">
+                    <div id="durationScoreScatter" class="scatter"></div>
+                    <script>
+                      $(function () {
+                        const xdata = [
+                          ${unsafeHtml(
+                            `${userScores.map((user) => {
+                              return user.duration_secs;
+                            })}`,
+                          )},
+                        ];
+                        const ydata = [
+                          ${unsafeHtml(
+                            `${userScores.map((user) => {
+                              return user.score_perc;
+                            })}`,
+                          )},
+                        ];
```

Comment:
Same comment as above.

Reply:
These have been moved to data attributes in [fe614f8](https://github.com/PrairieLearn/PrairieLearn/pull/9809/commits/fe614f896046789604dac2796569dc2b7421cebc).